src/lib.rs

```diff
@@ -1104,7 +1112,7 @@ impl<N: Name> Context<N> {
     /// assert_eq!(t.apply(&ctx2).to_string(), "bool → t1");
     /// // ctx2 uses t0 and t1
     ///
-    /// let ctx_change = ctx.merge(ctx2);
+    /// let ctx_change = ctx.merge(ctx2, vec![]);
```

Comment:
Lets add a separate example that uses `sacred`

Reply:
Done!